generators/apiref.fsx

```diff
@@ -13,6 +13,12 @@ open FSharp.MetadataFormat
 open Html
 open Apirefloader
 
+let stripMicrosoft (str: string) =
+    if (str.StartsWith("Microsoft")) then
+        str.Split('.').[1..] |> String.concat "."
```

Comment:
trollface suggestion: substring from "Microsoft".length to the end?

Reply:
pft